ext/drx/drx.c

```diff
@@ -71,7 +71,7 @@
 #    define IF_WINDOWS_ELSE(x, y) (y)
 #endif
 
-#if defined(X86) || defined(AARCH64)
+#if (defined(X86) || defined(AARCH64)) && !(defined(MACOS) && defined(AARCH64))
```

Comment:
Add an `XXX i#5383: ` comment specifying that PLATFORM_SUPPORTS_SCATTER_GATHER is still broken for MACOS in AARCH64.